bot/exts/evergreen/trivia_quiz.py

```diff
@@ -151,19 +379,22 @@ def check(m: discord.Message) -> bool:
 
                     response = random.choice(WRONG_ANS_RESPONSE)
                     await ctx.send(response)
-                    await self.send_answer(ctx.channel, question_dict)
+                    await self.send_answer(ctx.channel, answers, question_dict)
                     await asyncio.sleep(1)
 
                     hint_no = 0  # init hint_no = 0 so that 2 hints/time alerts can be sent for the new question.
 
-                    await self.send_score(ctx.channel, self.game_player_scores[ctx.channel.id])
+                    await self.send_score(
```

Comment:
another instance of black formatting not necessarily being better

Reply:
resolved